
  exploration_planner_node:
    ros__parameters:
      max_distance_to_turn: 0.4 #distance in meters in front of the robot before it starts to change direction [m]
      motion_command_frequency: 5.0 #how many times per second motion comand will be sent to motion_controller
      inf_weight: 0.05 #how much impact on turning infinite values from sensor have
      min_distance_weight: 1.0 #how much impact on turning minimum range values have
      side_angle_range: 45 #angle of view on the left and right sides which robot checks to make decision about turning

  object_tracking_planner_node:
    ros__parameters:
      object_name: 'nike_ball' #name of the detected object
      not_found_name: 'not_found' #value when object is not detected
      motion_command_frequency: 5.0 #how many times per second motion comand will be sent to motion_controller
      image_height: 480.0 
      image_width: 640.0 
  
  collision_avoidance_planner_node:
    ros__parameters:
      distance_from_obstacle_treshold: 0.4 #distance from the obstscle to set collision state [m]
      backtracking_enabled: true #disable or enable backwards movement when collision is detected 
      backtracking_max_time: 10 #maximum time of going back in if backtracking is enabled
      motion_command_frequency: 5.0 #how many times per second motion comand will be sent to motion_controller
      sleep_after_collision_time: 3 #how much time robot will wait after a collsion was detected

  motion_controller_node:
    ros__parameters:
      max_linear_speed: 0.3 #maximum linear speed of the robot [m/s]
      min_linear_speed: -0.3 #minimum linear speed of the robot [m/s]
      max_rotation_speed: 1.2 #maximum rotational speed [rad/s]
      min_rotation_speed: -1.2 #minimum rotational speed [rad/s]
      min_turning_radius: 0.2 #minimum turning radius [m]
      rotational_acceleration: 0.1 #rotational acceleration [rad/s^2]
      controller_frequency: 10.0 #how many times per second cmd_vel message is published [1/s]